.github/workflows/auto-publish.yml

```diff
@@ -19,6 +19,7 @@ on:
   push:
     branches:
       - master
+  workflow_dispatch: {}
```

Comment:
We do not want the push flow to be run manually.
If something is that broken that a publishing action to the production site needs to be run manually then we need to run it fully manually.